src-tauri/src/utils/resolve.rs

```diff
@@ -252,9 +252,33 @@ pub async fn create_window(is_show: bool) -> bool {
 
     if !is_show {
         logging!(info, Type::Window, true, "静默模式启动时不创建窗口");
-        lightweight::set_lightweight_mode(true).await;
-        handle::Handle::notify_startup_completed();
-        return false;
+        // 只有在启用静默启动且启用自动轻量模式时才设置轻量模式
+        let verge_config = Config::verge().await;
+        let enable_silent_start = verge_config
+            .latest_ref()
+            .enable_silent_start
+            .unwrap_or(false);
+        let enable_auto_lightweight = verge_config
+            .latest_ref()
+            .enable_auto_light_weight_mode
+            .unwrap_or(false);
+
+        if enable_silent_start && enable_auto_lightweight {
+            logging!(
+                info,
+                Type::Window,
+                true,
+                "静默启动且开启自动轻量模式，设置轻量模式"
+            );
+            lightweight::set_lightweight_mode(true).await;
```

Comment:
```suggestion
            lightweight::set_lightweight_mode(true).await;
            handle::Handle::notify_startup_completed();
            return false;
```

原有的逻辑除了设置轻量模式，还会通知应用已经启动完成并快速返回一个 `false` 以结束函数。

为了保持这个行为，或许还是应该加回去？